src/index.ts

```diff
@@ -0,0 +1,5 @@
+import usePromise from './usePromise'
+
+export * from './usePromise'
+export { usePromise }
```

Comment:
Not a fan of including another way to import the library, this only complicates the API surface unnecessarily. The fact that some IDEs (like WebStorm) cannot handle default imports properly, even though this method is clearly named 'usePromise' both in source and the type definitions should not influence the design of this library.

Reply:
Introduced the `index.ts` as a more generic pattern to provide the API, but as discussed, lets either convert `usePromise` to `index.ts` but let's indeed not make extra files